src/Common/Jemalloc.cpp

```diff
@@ -145,11 +145,14 @@ void jemallocDeallocationTracker(const void * ptr, unsigned usize)
     }
 }
 
-thread_local std::string last_flush_profile;
+thread_local std::array<char, 256> last_flush_profile_buffer;
+thread_local std::string_view last_flush_profile;
```

Comment:
@azat AFAIK this should be lazily allocated so we shouldn't end up with all the threads taking this space.
Can you confirm this?

Reply:
Yes, it should be done in lazy fashion.